scss/_variables.scss

```diff
@@ -249,9 +249,9 @@ $body-text-align:           null !default;
 // Style anchor elements.
 
 $link-color:                              $primary !default;
-$link-decoration:                         none !default;
+$link-decoration:                         underline !default;
 $link-hover-color:                        darken($link-color, 15%) !default;
-$link-hover-decoration:                   underline !default;
+$link-hover-decoration:                   $link-decoration !default;
```

Comment:
We could `null`ify this. This way we could also remove the hover text-decorations from buttons.